src/Promitor.Scraper.Tests.Unit/Serialization/v1/Core/AggregationDeserializerTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.ComponentModel;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Core;
+using Xunit;
+
+namespace Promitor.Scraper.Tests.Unit.Serialization.v1.Core
+{
+    [Category("Unit")]
+    public class AggregationDeserializerTests
+    {
```

Comment:
How about adding tests where the fields are not specified?

Reply:
Maybe do this for all deserializer tests wher it makes sense?

Reply:
Did I not? I think all the fields have a `Supplied` and `NotSupplied` test, so if you take a look at `Deserialize_IntervalNotSupplied_UsesDefault` it doesn't specify the interval and checks the behaviour.

Is that what you mean? If not can you give me an example?

Reply:
I don't fully recall but I think I was hinting at a test where the input is the following:
`@"aggregation:";` but you kindof do this already so that's fine!